.changeset/config.json

```diff
@@ -7,5 +7,11 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": [
```

Comment:
[MEMO] 
This is only a temporary fix, according the changeset's [documentation](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#ignore-array-of-packages). Each package should be marked private if we don't want to publish them.